static/js/reviewList.js

```diff
@@ -0,0 +1,99 @@
+window.addEventListener("load", onPageLoad);
+const currentPath = window.location.href;
+const movieID = currentPath.substring(currentPath.lastIndexOf("/") + 1);
+const buttonNextEl = document.querySelector("#buttonNext");
+const buttonPrevEl = document.querySelector("#buttonPrev");
+const reviewListEL = document.querySelector("#reviewList");
+buttonNextEl.addEventListener("click", getNextPage);
+buttonPrevEl.addEventListener("click", getPrevPage);
+let currentPage = 1;
+let totalPages = 2;
+
+async function getNextPage() {
+	if (currentPage != totalPages) {
+		currentPage++;
+	}
+	const reviews = await getReviewsFromAPI(movieID);
+	insertReviewsInHTML(reviews);
+	disableButtons();
+}
+
+async function getPrevPage() {
+	if (currentPage != 1) {
+		currentPage--;
+	}
+	const reviews = await getReviewsFromAPI(movieID);
+	insertReviewsInHTML(reviews);
+	disableButtons();
+}
+
+function disableButtons() {
+	if (currentPage == 1) {
+		buttonPrevEl.disabled = true;
+	} else {
+		buttonPrevEl.disabled = false;
+	}
+	if (currentPage == totalPages) {
+		buttonNextEl.disabled = true;
+	} else {
+		buttonNextEl.disabled = false;
+	}
+}
+
+async function onPageLoad() {
+	const reviews = await getReviewsFromAPI(movieID);
+	console.log(reviews);
```

Comment:
I think you forgot to remove a bugg hunting console.log :)

Reply:
Nice catch! 🪲